src/database/schema/ChatSchema.ts

```diff
@@ -266,6 +304,7 @@ class ChatSchema implements BaseDbSchema {
     userId: string,
     channelId: string,
     message: string,
+    attachments: any,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

As mentioned before, avoid using `any` type. Use a specific type or interface for `attachments`.
```diff
-    attachments: any,
+    attachments: AttachmentInterface[], // Define this interface according to your needs
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->